### PR TITLE
adds responseText to installmentBuilder for plan

### PR DESCRIPTION
**Pull request by rommelfreddy**

### CODE REVIEW

File: src/Frontend/InstallmentBuilder.php [hunk 1]
```diff
@@ -340,7 +340,11 @@ public function getInstallmentPlanAsJson($amount, $type, $value)
     {
         $configuration = $this->getInstallmentCalculation($type, $value, $amount);
 
-        return json_encode($configuration->getResult());
+        $result = $configuration->getResult();
+        $rpReasonCodeTranslation = 'rp_reason_code_translation_' . $configuration->getReasonCode();
+        $result['responseText'] = $this->lang->$rpReasonCodeTranslation();
```
Comment (eiriarte-mendez) [inline src/Frontend/InstallmentBuilder.php:345]: this call will throw an exception if reason code is not-valid/unknown/empty...

**Review by eiriarte-mendez — COMMENTED** [review]
dangerous call?

File: src/Frontend/InstallmentBuilder.php (hunk 1 above)
Comment (rommelfreddy) [reply src/Frontend/InstallmentBuilder.php:345]: just got this from here: https://github.com/ratepay/php-library/pull/9/files/b68170d7f002921e30be0d6698c6518eb9233917#diff-20a38d9fb28d7ce882a4cbf553184954R322 

a few lines earlier ...

**Review by eiriarte-mendez — APPROVED** [review]
works

**Review by hamail-tahir — APPROVED** [review]

**Pull request merged**